tensilelite/Tensile/Source/lib/include/Tensile/SingleSolutionLibrary.hpp

```diff
@@ -92,18 +92,7 @@ namespace TensileLite
 
             if(solution)
             {
-                if(debug)
-                {
-                    std::cout << "hardwarePredicate:" << std::endl;
-                    solution->hardwarePredicate->debugEval(hardware, std::cout);
-                    std::cout << std::endl;
-                    std::cout << "problemPredicate:" << std::endl;
-                    solution->problemPredicate->debugEval(problem, std::cout);
-                    std::cout << std::endl;
```

Comment:
The debug output in canSolve looks a bit different than the old code. Did you check if the debug output is still readable?

Reply:
good catch, I've added it back